packages/oauth/oauth-provider/src/token/token-manager.ts

```diff
@@ -272,12 +261,6 @@ export class TokenManager {
       sub: account.sub,
     }
 
-    await this.hooks.onTokenResponse?.call(null, tokenResponse, {
-      client,
-      parameters,
-      account,
-    })
```

Comment:
Any special reason this hook was removed?

Reply:
Initially I wanted the `oauth-provider` to be as generic as possible and added hooks wherever it made sense to add atproto specific logic. Then we agreed that this was not a necessity. So now, I try to remove unused code where applicable.